app/assets/javascripts/components/StickyColumn.js

```diff
@@ -0,0 +1,215 @@
+define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, DoughBaseComponent, eventsWithPromises) {
+  'use strict';
+
+  var StickyColumn,
+      defaultConfig = {
+        selectors: {
+          mainContent: '.l-article-3col-main',
+          parent: '.l-article-3col-right'
+        },
+        classes: {
+          fixed: 'related-links--desktop-fixed',
+          bottom: 'related-links--desktop-bottom'
+        }
+      };
+
+  StickyColumn = function($el, config) {
+    StickyColumn.baseConstructor.call(this, $el, config, defaultConfig);
+
+    this.contentHeight = 0;
+    this.topMargin = 0;
+    this.top = 0;
+    this.bottom = 0;
+    this.isFixed = false;
+    this.isAtBottom = false;
+    this.isInSidebar = false;
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(StickyColumn);
+
+  /**
+   * Initialize the component
+   * @param {Promise} initialised
+   * @return {StickyColumn}
+   */
+  StickyColumn.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+
+    this._bindEvents();
+    this._measureDom();
+    this._showElement();
+    this._positionComponent();
+
+    return this;
+  };
+
+  /**
+   * setup resize and scroll events to handle change in dimensions
+   */
+  StickyColumn.prototype._bindEvents = function() {
+    $(window)
+    .on('resize', this._debounce($.proxy(this._handleResize, this), 200))
+    .on('scroll', $.proxy(this._handleScroll, this));
+
+    eventsWithPromises.subscribe('toggler:toggled', $.proxy(this._handleSectionToggle, this));
+  };
+
+  /**
+   * Test whether the element is visible on the page
+   * @return {Boolean}
+   */
+  StickyColumn.prototype._isElementVisible = function(el) {
```

Comment:
Its implied from the `.is` but might tend to lean towards using `$el` for jQuery collections